deltachat-rpc-server/src/bin/deltachat-rpc-server/main.rs

```diff
@@ -11,18 +11,24 @@ use deltachat_jsonrpc::api::{Accounts, CommandApi};
 use futures_lite::stream::StreamExt;
 use tokio::io::{self, AsyncBufReadExt, BufReader};
 use tokio::task::JoinHandle;
+use tracing::{info, trace};
```

Comment:
Maybe we can reexport all such macros somewhere in our crate to just write smth like `use crate::tracing::macros::*`? Because if we need, say, `info`, we also need other macros too and it's not convenient to constantly fix these `use` statements